app/views/static/english_language_guidance.html.erb

```diff
@@ -0,0 +1,91 @@
+<% content_for :page_title, 'English language guidance' %>
+<% content_for :back_link_url, back_link_url %>
+
+<h1 class="govuk-heading-xl">Use an English language test in your qualified teacher status (QTS) application</h1>
+
+<p class="govuk-body-lead">If you plan to provide evidence that you’ve passed an English language test as part of your QTS application, you’ll need to make sure your test meets all the requirements.</p>
+
+<h2 class="govuk-heading-m">What your English language test must show</h2>
+
+<p class="govuk-body">For QTS, the test must be a ‘Secure English Language test’ (SELT). That means it was taken:</p>
+
+
+<ul class="govuk-list govuk-list--bullet">
+    <li>with an approved provider</li>
+    <li>at an approved test location</li>
+    <li>within the last 2 years</li>
+</ul>
+
+<p class="govuk-body">Your test must be at level B2 or above of the <a
+  href="https://www.coe.int/en/web/common-european-framework-reference-languages/level-descriptions"
+  class="govuk-link">Common European Framework of Reference for Languages (CEFR)</a>.
+</p>
+<h3 class="govuk-heading-m">Approved providers and tests</h3>
+
+<p class="govuk-body">The Teaching Regulation Authority (TRA) recognises 5 approved SELT providers for QTS.</p>
+
+<p class="govuk-body">
+  Each provider offers a number of different tests, so it’s important to understand which ones we can accept.
+</p>
+
+<p class="govuk-body">
+  You’ll also need to know your provider’s required scores for the 4 disciplines of listening, speaking, reading and writing.
+</p>
+
+<table class="govuk-table">
```

Comment:
Could we use the `govuk-components` to render the table rather than the raw HTML?

https://govuk-components.netlify.app/components/table/#input-erb-building-a-table-row-by-row

This approach works, but using the component means it'll automatically keep up to date if the styles or markup of the GOV.UK Design System changes.